### ISSUE TITLE
Not Shown DeFi Tokens on DeFi Section

### ISSUE THREAD

**Issue by Hoanib** [AlphaWallet/alpha-wallet-ios#4722]
Just a few tokens are displayed on the new DeFi Section. For example CVX (a DeFi token) is displayed but CRV is not. All my bTokens deposited on DeFi are not displayed. The BadgerDAO token (a DeFi governance token) is not displayed and it isn't displayed on the Governance Section. 

DeFi Section example:

![image](https://user-images.githubusercontent.com/100842739/171293510-bacb42e1-ed83-4f57-b321-5860c772ba3b.png)

All Section example:

![image](https://user-images.githubusercontent.com/100842739/171293797-4d78811a-62e9-423f-9e01-6f68bbff9803.png)

![image](https://user-images.githubusercontent.com/100842739/171293819-b4ad2b7e-5cf2-4f78-8e7e-fdbb875b0ec1.png)

**Comment by Hoanib** [comment]
If it's needed, a list of DeFi token contract addresses not displayed:

BadgerDAO: 0x3472a5a71965499acd81997a54bba8d852c6e53d

Vault Address Badger Sett Curve.fi Factory BTC Metapool: ibBTC 
bibbtc/sbtcCRV-f: 0xaE96fF08771a109dc6650a1BdCa62F2d558E40af

Underlying LP Tokens Curve.fi Factory BTC Metapool: ibBTC 
ibBTC/sbtcCRV-f: 
0xFbdCA68601f835b27790D98bbb8eC7f05FDEaA9B

bCVXCRV: 0x2b5455aac8d64c14786c3a29858e43b5945819c0

bveCVX: 0xfd05D3C7fe2924020620A8bE4961bBaA747e6305

bbveCVX: 0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551

Curve DAO (CRV): 0xD533a949740bb3306d119CC777fa900bA034cd52

Vault Address Badger Sett Curve.fi renBTC/wBTC
bcrvRenWBTC: 0x6dEf55d2e18486B9dDfaA075bc4e4EE0B28c1545

bCVX: 0x53c8e199eb2cb7c01543c137078a038937a68e40

**Comment by hboon** [comment]
@Hoanib thanks the list is great. The groups (where the tokens appear) other than then in "All" is controlled by a file we build in this repo: https://github.com/AlphaWallet/tokens/. We can't add tokens directly to that repo at the moment, but created an issue to track: https://github.com/AlphaWallet/tokens/issues/4

**Comment by Hoanib** [comment]
@hboon, Check this dApp (https://debank.com/) All DeFi tokens are registered in all chains and also the dApp registers the total balance by chain and of all chains.

**Comment by hboon** [comment]
Yeah, aware of that. They have an API I intend to look at. It's at least useful for the total balance.
